release-notes/aws/10.0.0-rc1

```diff
@@ -0,0 +1,29 @@
+## :zap:  Giant Swarm Release 10.0.0-rc1 for AWS is now active for you! :zap:
+
+We are pleased to announce that Node Pools is now released! 
+
+Please see here the supported material which we recomend reading before getting started: https://docs.giantswarm.io/basics/nodepools
+
+We also kindly remind you that from this release and for a period of SIX months onwards, to give you a chance to move your clusters, we will be supporting both the 'Legacy' and the 'New' branches separately which will include:
+
+:older_adult: 'Legacy' Branch: 
+
+pre-NodePools releases including 9.x/ k8s 1.15
+security patches
+critical bug fixes to existing functionality
+updates to Managed Apps such as the Ingress Controller (with the exception of features that cannot be back-ported without significant invest)
+Plus, the upcoming k8s 1.16 release 
+
+Also, please note,
+there will be no direct upgrade path from release 9.x to 10.x and you will need to move your cluster
+the six month window for this will start on the date of the NodePool release and run until approximately mid-June. We will give you plenty of prior warning. 
+
+:baby: 'New' Branch: 
```

Comment:
As @puja108 already mentioned. We usually have way more details about the release. 

See an example here: https://github.com/giantswarm/installations/pull/794

Reply:
yep thank you.. Marcel's going to help me with these details ...